database/db.js

```diff
@@ -0,0 +1,3 @@
+const pgp = require('pg-promise')
+const connectionString = 'postgres://localhost:5432/funnybone'
```

Comment:
Consider retrieving `connectionString` from a `process.env` setting to allow for running in a production environment.